### PR TITLE
Added document retrieval for generation

### PR DESCRIPTION
**Pull request by alistairknight**
# **What this PR does / why we need it**:
*A brief description of changes being made*
*Link to the relevant work items: e.g: Relates to ADO Work Item AB#213700 and builds on #3376 (link to ADO Build ID URL)*

# **Special notes for your reviewer**
*Any specific actions or notes on review?*

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [ ] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [ ] Title pattern should be `{work item number}: {title}`
- [ ] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests
- [ ] Version number in **package.json** has been updated

# **How does this PR make you feel**:
gif

### CODE REVIEW

**Review by BarryGeeDefra — APPROVED** [review]
LGTM

**Review by BarryGeeDefra — DISMISSED** [review]
removing approval breaks on local build.

File: app/base.js
Comment (BarryGeeDefra) [inline app/base.js:1]: duplicate file needs removed.

File: app/config/api.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  documentsApi: Joi.object({
+    baseUrl: Joi.string().required()
+  }),
+  responsesApi: Joi.object({
```
Comment (BarryGeeDefra) [inline app/config/api.js]: this service isnt used. causing crash on local build.

**Review by BarryGeeDefra — COMMENTED** [review]
couple of things to review.

File: app/base.js
Comment (alistairknight) [reply app/base.js:1]: Removed

File: app/config/api.js (hunk 1 above)
Comment (alistairknight) [reply app/config/api.js]: Removed

**Review by sfitz42 — APPROVED** [review]
LGTM

**Review by BarryGeeDefra — APPROVED** [review]
LGTM

**Review by sebhoron — APPROVED** [review]
LGTM

**Pull request merged**